Woodoku/src/UI/CommandLineUI.py

```diff
@@ -0,0 +1,30 @@
+from abc import ABC
+from typing import List, Tuple
+
+from Woodoku.src.Entity.WoodokuBoard import WoodokuBoard
+from Woodoku.src.Entity.WoodokuShape import WoodokuShape
+from Woodoku.src.UI.InterfaceUI import InterfaceUI
+
+
+class CommandLineUI(ABC, InterfaceUI):
```

Comment:
```suggestion
class CommandLineUI(InterfaceUI):
```

Repeated inheritance, `InterfaceUI` already inherits from `ABC`

Reply:
Just updated!